packages/theme-chalk/src/table-v2.scss

```diff
@@ -130,7 +130,7 @@
     right: 0;
     top: 0;
     bottom: 0;
-    z-index: 9999;
+    z-index: 99;
```

Comment:
## Suggest

Hi! Looking at this problem, there was a common problem not only in the table but also in the components related to Feedback > Loading.

Therefore, it would be better to change the z-index of the Loading component globally or change the z-index of the algolia modal to reduce side effects.

What do you think?

![image](https://user-images.githubusercontent.com/27342882/184275630-875984ff-08ec-42ea-8cd7-38723c3b39e8.png)

Reply:
Thanks for your suggestion, I think it is very good to change the z-index of algolia modal ❤

Reply:
How about the method below? There is no style scoped, so there is no problem with execution.

```vue
// vitepress/components/navbar/vp-search.vue

<script></script>
<template>
  <div id="docsearch" class="algolia-search-box" />
</template>
<style lang="scss">
.DocSearch {
  // (...)
  &.DocSearch-Container { // here
    z-index: 20000;
  }
  &.DocSearch-Button {
    margin-right: 8px;
  }
  // (...)
}
</style>
```

Reply:
Thanks for the clear code, I'll update it.